src/components/Modal/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Aux from '../../hoc/Auxiliary';
+import Backdrop from '../Backdrop';
+
+const Modal = styled.div`
+  position: fixed;
+  z-index: 500;
+  background-color: white;
+  width: 70%;
+  border: 1px solid #ccc;
+  box-shadow: 1px 1px 1px black;
+  padding: 16px;
+  left: 15%;
+  top: 30%;
+  box-sizing: border-box;
+  transition: all 0.3s ease-out;
+
+  @media (min-width: 600px) {
+    width: 500px;
+    left: calc(50% - 250px);
+  }
+`;
+
+export default React.memo((props) => {
```

Comment:
why memo? sure you need it?

Reply:
Removed memo for now. I used it to reduce the number of times the modal renders to only when we want to show it.